### PR TITLE
fix: code is now in pep8

### PR DESCRIPTION
**Pull request by JakubNvk**
To: @mrshu 
Cc: @Adman

### CODE REVIEW

File: sktravel.py [hunk 1]
```diff
@@ -17,13 +16,14 @@
...
 def split_args_by(args, by):
+    """Split string by character <by>."""
     return list(map(lambda x: x.strip(), args.split(by)))
 
 
 class Travel(BotPlugin):
 
     def send_output(self, nick, dep, dest, date, result=None):
-        """Sends output"""
+        """Send output."""
```
Comment (mrshu) [inline sktravel.py:26]: How is this PEP8-related?

Comment (JakubNvk) [inline sktravel.py:26]: https://www.python.org/dev/peps/pep-0008/#introduction
https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

**Comment by Krista** [comment]
Another time, please name your pull request with respect to your work in code. I'm ok with your changes. @mrshu Can I close it now?

**Comment by mrshu** [comment]
@Krista sure, feel free to hit the big green Merge button -- this is your playground =)

**Pull request merged**